### PR TITLE
chore: add README content

### PR DESCRIPTION
**Pull request by imaginaris**
# Description
Added README content.
Renamed public API class.

# Checklist
- [X] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,59 @@
...
+1. `signature: String` - Signature to be verified encoded in base64.
+2. `keyId: String` - ID of public key to be fetched.
+3. `data: Data` - Data to be verified.
+4. `resultHandler: (_ verified: Bool) -> Void` - Handler called when verification is complete with `verified` representing verification status.
+
+Example of usage:
+```swift
+let data = "{"signed": "data"}".data(using: .utf8)!
+RSignatureVerifier.verify(signature: "YSBzaWduYXR1cmUK",
+                         keyId: "aac02fc3-1111-1111-1111-22baf9c5a281",
```
Comment (donnie-jp) [inline README.md:53]: Is the above sig and keyId dummy data?

File: Tests/Integration/IntegrationTests.swift [hunk 2]
```diff
@@ -35,7 +35,7 @@ class IntegrationSpec: QuickSpec {
                             return
                         }
 
-                        SignatureVerifier.verify(signature: bundleSignature,
+                        RSignatureVerifier.verify(signature: bundleSignature,
```
Comment (donnie-jp) [inline Tests/Integration/IntegrationTests.swift:38]: Is the alignment now off-by-1 below this line?

File: README.md (hunk 1 above)
Comment (imaginaris) [reply README.md:53]: Yes. you can try to decode signature 😉

File: Tests/Integration/IntegrationTests.swift (hunk 2 above)
Comment (imaginaris) [reply Tests/Integration/IntegrationTests.swift:38]: Yes 😞

Comment (imaginaris) [reply Tests/Integration/IntegrationTests.swift:38]: Fixed

**Review by donnie-jp — APPROVED** [review]

**Pull request merged**